cmd/localapiservice/localapishim.go

```diff
@@ -47,6 +48,13 @@ func Java_com_tailscale_ipn_ui_localapi_LocalApiClient_doRequest(
 	jbody C.jbyteArray,
 	jcookie C.jstring) {
 
+	defer func() {
```

Comment:
This helped me debug the panic below. Seems worth leaving in in case we encounter other panics.